### PR TITLE
Retrieve latest quacs-rs, quacs-data on build instead of relying on submodules

### PR DESCRIPTION
**Pull request by johnnyapol**

### CODE REVIEW

File: build_wasm.sh [hunk 1]
```diff
@@ -16,6 +16,10 @@ if test -z "$DONT_BENCHMARK_QUACS"; then
 	FLAGS="$FLAGS --features benchmark"
 fi
 
-git submodule update --init --remote --recursive
-cd src/quacs-rs
+# Update our local dependencies (quacs-rs, quacs-data), or clone if possible
+git -C src/store/data pull || git clone https://github.com/quacs/quacs-data src/store/data
+git -C src/quacs-rs pull || git clone https://github.com/quacs/quacs-rs src/quacs-rs
```
Comment (benjaminrsherman) [inline build_wasm.sh:21]: `quacs-rs` also has a dependency on `quacs-data`, so there should be a pull/clone for `src/quacs-rs/src/data` too.

Comment (benjaminrsherman) [reply build_wasm.sh:21]: Nevermind I didn't read the next line

File: package.json [hunk 2]
```diff
@@ -123,9 +123,9 @@
     "not dead"
   ],
   "gitHooks": {
-    "post-checkout": "git submodule update --init --remote --recursive && yarn install",
-    "post-merge": "git submodule update --init --remote --recursive && yarn install",
-    "pre-commit": "git submodule update --init --remote --recursive && git add src/quacs-rs src/store/data && yarn install && lint-staged"
+    "post-checkout": "yarn install",
```
Comment (benjaminrsherman) [inline package.json:126]: We weren't doing this before, but this should be `yarn install --frozen-lockfile` I think.

Comment (johnnyapol) [reply package.json]: Updated with latest push

**Pull request merged**